File: Java/ExerciciosJava/demo/src/main/java/com/exercicios/Exercicio5.java
//Algoritmo que converte metros para centímetros.
package com.exercicios;

import java.util.Scanner;

public class Exercicio5 {
    public static void main(String[] args) {
        //Variáveis
        float metros, centimetros;
        Scanner entrada = new Scanner(System.in);

        //Entrada
        System.out.println("Escreva um valor em metros: ");
        metros = entrada.nextFloat();

        //Processamento
        centimetros = (metros * 100);

        //Saída
        System.out.println(metros + " metros em centímetros é: " + centimetros);
        
        entrada.close();
    }
}

File: C/Exercicio5.c
//Algoritmo que converte metros para centímetros.
#include <stdio.h>

int main() {
    //Variáveis
    int metros, centimetros;

    //Entrada
    printf("Insira um valor em metros: \n");
    scanf("%d", &metros);

    //Processamento
    centimetros = (metros * 100);

    //Saída
    printf("%d metros convertido para centímetros equivale a: %d\n", metros, centimetros);
    return 0;
}